Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -52,10 +52,10 @@
     - state: icon-syndicate
   - type: ItemBorgModule
     items:
-    - DL6902Borg
-    # - WeaponLauncherChinaLakeBorg
+    - LaserMinigunBorg
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 5

(indentation)